src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -454,12 +457,31 @@ public static boolean doesEventValueHaveTimeStates(Class<? extends Event> event,
 		return getEventValueConverter(event, c, TIME_PAST, false) != null || getEventValueConverter(event, c, TIME_FUTURE, false) != null;
 	}
 
-	private record EventValueInfo<E extends Event, T>(
+	/**
+	 * @return All the event values for each registered event's class.
+	 */
+	public static Multimap<Class<? extends Event>, EventValueInfo<?, ?>> getPerEventEventValues() {
+		Multimap<Class<? extends Event>, EventValueInfo<?, ?>> eventValues = MultimapBuilder
+			.hashKeys()
+			.hashSetValues()
+			.build();
```

Comment:
You can just use HashMultimap.build() if you aren't needing array list values. The default of HashMultimap should already be sets. Also see ArrayListHashMap if you are needing array list values.

Reply:
this has to be present for you to be able to call build